modules/sbo-deploying-a-postgresql-database-operator-power-z.adoc

```diff
@@ -28,7 +28,9 @@ metadata:
   namespace: openshift-marketplace
 spec:
   sourceType: grpc
-  image: quay.io/ibm/operator-registry-<architecture> <1>
+  grpcPodConfig:
+    securityContextConfig: <security_mode> <1>
+  image: quay.io/ibm/operator-registry-<architecture> <2>
```

Comment:
need to add update the footnote links for these.

Reply:
Since this content is owned by SBO and the [registry images](https://github.com/openshift/openshift-docs/pull/53795/files#diff-b11a9d429a017c54cb0b7bd4b79c7d6e004b3edcfbc7f151134fd46eabe677bbL67-L68) use 4.9 and 4.8, I am going to remove reference to the security mode. I removed the footnote but forgot to scrub the code callout. I will reach out to the writer for SBO after 4.12 GA and let them know that this content might need to be updated.